cross/libmysqlclient/Makefile

```
@@ -7,7 +7,7 @@ PKG_DIST_NAME = $(DIST_NAME).$(PKG_EXT)
 PKG_DIST_SITE = https://cdn.mysql.com/archives/$(PKG_REAL_NAME)
 PKG_DIR = $(DIST_NAME)
 
-DEPENDS = native/libmysqlclient
+DEPENDS = native/libmysqlclient cross/openssl cross/zlib
```

we should always use BUILD_DEPENDS for native packages as those are built first with disabled parallel make and output is normally not included in the spk files.

```suggestion
BUILD_DEPENDS = native/libmysqlclient
DEPENDS = cross/openssl cross/zlib
```

PS
it was me, who created this package with DEPENDS instead of BUILD_DEPENDS ...

---

good catch!